src/main/java/Entities/UserPublicProfile.java

```diff
@@ -1,11 +1,40 @@
 package Entities;
-
 import java.io.Serializable;
 import java.util.HashMap;
-
 public class UserPublicProfile implements Serializable {
-//    private json;
+    private String biography;
+    private String coursePreferences;
+    private HashMap<String, String> preferences;
     public UserPublicProfile(){
-//        this.preferences = HashMap<String, String> ;
+        HashMap<String, String> preferences = new HashMap<>();
+        /*Prefilling in the preferences*/
+        preferences.put("Time Commitment", "");
+        preferences.put("Location", "");
+        preferences.put("Meeting Time", "");
+        this.preferences = preferences;
+    }
+
+    public String getCoursePreferences() {
```

Comment:
Thank you for implementing .getCoursePreferences(). I will adjust my code with respect to the new PublicProfile.